docs/dev.md

```diff
@@ -79,15 +79,13 @@ Integration tests can read the following system properties to obtain information
 
 #### devc, Container Mode
 
-The following is a technology preview. The features and parameters described below may change in future milestones or releases of the Liberty Maven plugin.
+Start a Liberty server in a local container using the Dockerfile that you provide. An alternative to the `devc` goal is to specify the `dev` goal with the `-Dcontainer` option.
 
-Start a Liberty server in a local container using the Dockerfile that you provide. An alternative to the `devc` goal is to specify the `dev` goal with the `-Dcontainer` option. 
+You need to install the Docker runtime (e.g. Docker Desktop) locally to use this Maven goal. The installed Docker Engine version must be 18.03.0 or higher.
 
-You need to install the Docker runtime (e.g. Docker Desktop) locally to use this Maven goal. You can examine the commands used to build and run the container by viewing the console output of dev mode.
+When dev mode runs with container support, it builds a container image and runs the container. You can examine the commands that it uses to build and run the container by viewing the console output of dev mode. Additionally, it still provides the same features as the `dev` goal. It monitors files for changes and runs tests either automatically or on demand. This mode also allows you to attach a debugger to work on your application. You can review the logs generated by your server in the Liberty directory in your project e.g. target/liberty/wlp/usr/servers/defaultServer/logs.
```

Comment:
Same comment as on the gradle documentation.

Reply:
Updated